.github/workflows/antithesis-test.yml

```diff
@@ -0,0 +1,110 @@
+---
+name: Build and trigger Antithesis exploration
+
+on:
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: "0 0 * * *" # run every day at midnight
```

Comment:
This is what Marek is suggesting to remove, jfyi :)

```suggestion
```

Reply:
Got you! Thanks!

Reply:
Done and added comment